ingest/src/main/java/com/microsoft/azure/kusto/ingest/source/AbstractSourceInfo.java

```diff
@@ -22,9 +20,20 @@ public void setSourceId(UUID sourceId) {
         this.sourceId = sourceId;
     }
 
+    // An estimation of the raw (uncompressed, un-indexed) size of the data, for binary formatted files - use only if known
+    private long rawSizeInBytes;
+
+    public long getRawSizeInBytes() {
+        return rawSizeInBytes;
+    }
+
+    // An estimation of the raw (uncompressed, un-indexed) size of the data, for binary formatted files - use only if known
+    public void setRawSizeInBytes(long rawSizeInBytes) {
```

Comment:
The fact that there is no difference between "raw size is 0" and "raw size is unknown" makes me a bit worried